### DISCUSSION TITLE
Show column not in DB

### DISCUSSION THREAD

**Discussion by dfioravanti** [aminalaee/sqladmin#508] [Q&A]
Hey everyone, I was wondering if this project supports defining columns at runtime as `flask-admin` does. What I mean is that supposed we have a model like

```python
class Table(Base):
   id: int
   name: str
```

and I would like to show a column in the `List` page that is called `status` that would depend on the `name`. For example `status == True` if `active` is in `name`. With `flask-admin` all I need to do is define a formatter and then add that formatter to the `column_formatters` ditionary. Is it possible to do something similar in this project?

**Comment by aminalaee** [comment]
Will be available in https://github.com/aminalaee/sqladmin/pull/544
